src/js/components/Tools.jsx

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var React = require('react');
+
+// Splitting each sidebar into its column
+var Tools = React.createClass({
+  classNames: 'toolbar',
+  render: function() {
+    return (
+      <div className={this.classNames}>
+        <ul>
+          <li>RECT</li>
+          <li>SYMBOL</li>
+          <li>AREA</li>
+          <li>LINE</li>
+          <li>TEXT</li>
+          <li>|</li>
+          <li>EXPORT</li>
+          <li>|</li>
+          <li><i className="fa fa-file-o"></i> CLEAR</li>
+          <li><i className="fa fa-undo"></i> UNDO</li>
+          <li><i className="fa fa-repeat"></i> REDO</li>
```

Comment:
This is a total nit, but it feels a little weird to have the pipes drawn out in the HTML. We don't need to fix this now but I'd be inclined to make this a nested list with

- new UL containing each add mark button
- Export button
- New UL with each history buttons

and to have CSS handle the presentational dividiers